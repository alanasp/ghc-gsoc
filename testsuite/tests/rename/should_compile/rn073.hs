{-# OPTIONS_GHC -fwarn-warnings-deprecations #-}

-- Test deprecation of variable and data constructor exports

module ShouldCompile where

import Rn073_B(funcRn073_C) --deprecated
import Rn073_C(funcRn073_C) --non-deprecated
import Rn073_A(funcRn073_C) --deprecated

main = funcRn073_C --no warning because non-deprecated import declaration exists
