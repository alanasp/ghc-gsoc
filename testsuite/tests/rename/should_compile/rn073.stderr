rn073.hs:7:1: warning: [-Wdeprecations (in -Wdefault)]
    Deprecated: "Rn073_C shall be imported directly"

rn073.hs:9:1: warning: [-Wdeprecations (in -Wdefault)]
    Deprecated: "Rn073_C shall be imported directly"
